test/e2e/resources/pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Pipeline
+metadata:
+  name: $PIPELINE_NAME
+spec:
+  pipelineDisplayName: $PIPELINE_NAME
+  pipelineName: $PIPELINE_NAME
+  pipelineDefinition: '{"Version": "2020-12-01", "Metadata": {}, "Parameters": [{"Name": "ProcessingInstanceType", "Type": "String", "DefaultValue": "ml.m5.xlarge"}, {"Name": "ProcessingInstanceCount", "Type": "Integer", "DefaultValue": 1}, {"Name": "InputData", "Type": "String", "DefaultValue": "s3://$SAGEMAKER_DATA_BUCKET/pipeline-model-example/data/raw"}], "PipelineExperimentConfig": {"ExperimentName": {"Get": "Execution.PipelineName"}, "TrialName": {"Get": "Execution.PipelineExecutionId"}}, "Steps": [{"Name": "PreprocessData", "Type": "Processing", "Arguments": {"ProcessingResources": {"ClusterConfig": {"InstanceType": "ml.m5.large", "InstanceCount": {"Get": "Parameters.ProcessingInstanceCount"}, "VolumeSizeInGB": 30}}, "AppSpecification": {"ImageUri": "246618743249.dkr.ecr.us-west-2.amazonaws.com/sagemaker-scikit-learn:0.23-1-cpu-py3", "ContainerEntrypoint": ["python3", "/opt/ml/processing/input/code/preprocess.py"]}, 
```

Comment:
image hardcoded to us-west-2, will cause problem for canaries

also I see some dataset in 
{"Name": "InputData", "Type": "String", "DefaultValue": "s3://$SAGEMAKER_DATA_BUCKET/pipeline-model-example/data/raw"}

can we reuse the dataset from existing processing job test instead of adding more data? what is the size ?

Reply:
did some more work, replaced this with model-creation as previously discussed to not spend time or resources creating a processing/training job.